src/core.js

```diff
@@ -2080,7 +2080,7 @@ define([
                 properties: this.getMainProperties(),
                 help: {
                     title: "Basic",
-                    text: "<p>The <strong>Label</strong> is text that appears in the application. " +
+                    text: "<p>The <strong>Display Text</strong> appears in the application. " +
```

Comment:
This was an Amelia suggestion; apparently it used to be `Display Text` before it was `Label`. I don't have a strong preference for either - I don't think there's a good word to use here that'll convey the concept to everyone - but I do like getting rid of the "Display Text" placeholder.